### PR TITLE
Update to Replicache 6.2.0

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "singleQuote": false,
```
Comment (arv) [inline .prettierrc.json:2]: I had to add this file or I ended up with conflicting prettier rules which changed everything on save

File: frontend/data.ts [hunk 2]
```diff
@@ -27,7 +27,10 @@ import type { UserInfo } from "../shared/client-state";
 export type Data = Await<ReturnType<typeof createData>>;
 type Await<T> = T extends PromiseLike<infer U> ? U : T;
 
-export async function createData(rep: Replicache) {
+export async function createData(
+  rep: Replicache<typeof mutators>,
```
Comment (arv) [inline frontend/data.ts:31]: This worked out nicer than I first thought... I was like sh*t, this is not going to work... If you only do `Replicache` (without type param) it does not recognize the mutators.

File: frontend/data.ts [hunk 3]
```diff
@@ -37,106 +40,25 @@ export async function createData(rep: Replicache) {
...
-
-    deleteAllShapes: rep.register(
-      "deleteAllShapes",
-      async (tx: WriteTransaction) => {
-        await Promise.all(
-          (await tx.scanAll({ prefix: `shape-` })).map(([k]) => tx.del(k))
-        );
-      }
-    ),
+    ...rep.mutate,
```
Comment (arv) [inline frontend/data.ts:55]: I kept this on data since that is what we pass around

**Comment by arv** [comment]
TBR

File: frontend/data.ts (hunk 3 above)
Comment (aboodman) [reply frontend/data.ts:55]: I think we can get rid of `Data` now though. The only reason for it was to implement this pattern that `Replicache` now does natively.

File: frontend/data.ts (hunk 2 above)
Comment (aboodman) [reply frontend/data.ts:31]: Nice. Another way to do it is probably to capture the type of the constructed Replicache instance somewhere, kind of like how I was capturing the type of the returned `data` value.

**Pull request merged**